.github/workflows/changelog-check.yml

```diff
@@ -0,0 +1,38 @@
+on:
+  pull_request_target:
+    paths:
+      - 'aws/*'
+      - '!aws/*_test.go'  
+
+name: Changelog Check
```

Comment:
LGTM 👍 just curious, if all PRs will be opened w/o a changelog file b/c we don't initially know the PR#, we'll expect this comment to be on all new PRs? or do we expect enough delay time for a user to push up the changelog before the job gets triggered? i remember reading around about accepting changelog's w/o #'s but can't find if that's in place or will be

Reply:
cool, def not too much, serves as a good reminder 👍